x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/service_accounts/10_basic.yml

```diff
@@ -0,0 +1,88 @@
+---
+setup:
+  - skip:
+      features: headers
```

Comment:
Do we need to skip on the `headers` feature? We don't seem to use it.

Reply:
You are right. The copy/paste behaviour is showing through 🤦